src/app/mod.rs

```diff
@@ -111,5 +116,9 @@ impl App {
         if self.all || self.randomize_seed {
             randomize_seed::run(&world)?;
```

Comment:
This could become `set_seed::run(&world, rand::random())?;`

Reply:
True. I'll edit the PR.